Dockerfile

```diff
@@ -46,6 +46,9 @@ RUN apt-get update &&\
         python3-tk \
         srm-ifce-dev \
         wget \
+	git \
```

Comment:
I'm conflicted about adding `git` into the container. I've been avoiding it for two reasons.

1. Using it to download code means that there is a greater likelihood that the image build will depend on _when_ it is built unless care is taken to always checkout a specific version tag (which you have done below, just saying).
2. Including `git` in the container potentially gives users the impression that the container is _interactive_, which we want to discourage because the internal container environment is very fragile (i.e. users should be discouraged from opening a shell within the container).

I'm saying all this just to document my request to delete `git` and update the source-download below accordingly.

```suggestion
```

Reply:
I'll comment on this on the GENIE part below, but now removed.